### PR TITLE
Don't show get empty project message if project has monopods

### LINKED ISSUE #2355: Project is not empty, yet I am told it is
I have a project with pods I create on my own.

I go to the console and see the following:

si2bz6u

### PR DESCRIPTION
**Pull request by spadgett**
Fixes #2355

openshift_management_console

### CODE REVIEW

**Comment by spadgett** [comment]
@liggitt let me know if you think this is sufficient. I understand there are some edge cases with deployments I might not be catching, but you'll still see the "No services to show" message at least.

**Comment by spadgett** [comment]
Rebased.

openshift_management_console

**Comment by spadgett** [comment]
Hm.

```
!!! Error in hack/test-cmd.sh:550
  'osc delete all --all' exited with status 1
Call stack:
  1: hack/test-cmd.sh:550 main(...)
Exiting with status 1
2015/05/21 12:58:39 received terminated signal, shutting down...
[FAIL] !!!!! Test Failed !!!!
make: *** [check-test] Error 1
```

File: assets/app/scripts/controllers/overview.js [hunk 1]
```diff
@@ -354,6 +362,18 @@ angular.module('openshiftConsole')
       Logger.log("builds (subscribe)", $scope.builds);
     }));
 
+    // Show the "Get Started" message if the project is empty.
+    function updateShowGetStarted() {
+      // Check if there are any services, pods, or deployment configs.
+      var projectEmpty =
```
Comment (liggitt) [inline assets/app/scripts/controllers/overview.js]: reframe this as `hasContent` so you can short-circuit. `angular.equals` looks scary and expensive... wondering if `$filter('hashSize')(...) == 0` could work instead

**Comment by liggitt** [comment]
Also wondering if we should soften the "Your project is empty." language a bit... there are still things you could create in a project that would leave this page empty. Something welcoming but less emphatic about there being nothing in their project might be good if something springs to mind

**Comment by derekwaynecarr** [comment]
Your project has nothing running?

**Comment by spadgett** [comment]
@liggitt Switched to use `hashSizeFilter`.

Open to suggestions on the wording. What are filtering from the overview other than run-once pods?

**Comment by spadgett** [comment]
I wonder if we shouldn't just be showing run-once pods, since they're reaped after some time anyway.

**Comment by liggitt** [comment]
Deployment configs without deployments, people you've given roles to, image streams, templates, etc, all don't appear in the overview.

**Comment by spadgett** [comment]
@liggitt Anything left to do?

**Comment by liggitt** [comment]
this is fine for now, rebase

**Comment by spadgett** [comment]
@liggitt rebased

**Comment by liggitt** [comment]
[merge]

**Comment by liggitt** [comment]
```
W0527 19:01:12.726477   12667 status_manager.go:60] Failed to updated pod status: error updating status for pod "hello-openshift": pods "hello-openshift" not found
error: timed out waiting for the condition
!!! Error in hack/test-cmd.sh:573
  'osc delete all --all' exited with status 1
```
re[merge]

**Pull request merged**